src/commands/init/generateTemplate/index.js

```diff
@@ -54,7 +53,7 @@ export default function generateTemplate(name, src, dest, done) {
             }
             log.success('Project created');
             done();
-            logMessage(opts.completionMessage, data);
+            logMessage(opts.completionMessage, { data, ...metalsmith.metadata() });
```

Comment:
Ah, totally! Otherwise it just gets destructured as { data: data }. Nice catch, thx :)